### ISSUE TITLE
snapcraft: should we push the snap after it pipe runs

### ISSUE THREAD

**Issue by caarlos0** [goreleaser/goreleaser#680]
Should we push the file using `snapcraft push --release=stable file.snap` after the pipe runs?

**Comment by caarlos0** [comment]
working on it.

**Comment by caarlos0** [comment]
https://forum.snapcraft.io/t/requesting-approval-for-goreleasers-classic-snap/6571

**Comment by caarlos0** [comment]
if they approve, I'll add the feature to goreleaser to auto-upload every artifact :)

**Comment by mgrachev** [comment]
@caarlos0 Hi! I really need this feature. How can I help you with this?

**Comment by caarlos0** [comment]
I'm not sure yet how I would implement this... 

I actually think I need to split snapcraft into two pipes, one to build artifacts and other to push them, not sure how yet...

I'll probably work on this over the weekend, for now I'm not sure how to implement this...

**Comment by mgrachev** [comment]
I'd implement like Homebrew pipe with `skip_push` option.

**Comment by caarlos0** [comment]
yeah, but homebrew is the last step, snapcraft is in the middle...
